utils/bigint/Cargo.toml

```diff
@@ -8,6 +8,6 @@ edition = "2018"
 repository = "https://github.com/ChainSafe/forest"
 
 [dependencies]
-num-bigint = "0.3"
+num-bigint = { version = "0.3",  features = ["serde"] }
```

Comment:
nooo don't do this, this is the wrong serialization format and will trip up someone next if you leave it in

Reply:
lol thought it was weird we didnt use the serde feature for BigInt anywhere my bad